forge-game/src/main/java/forge/game/zone/MagicStack.java

```diff
@@ -462,10 +469,6 @@ private SpellAbilityStackInstance push(final SpellAbility sp) {
 
         GameActionUtil.checkStaticAfterPaying(sp.getHostCard());
```

Comment:
you should check if the LKI needs to be done after this StaticCheck?

Reply:
This is still the case because push() jumps back to add() and nothing interesting happens between

Though probably nothing would care about either order